custom-connectors/secplugs/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@Ajaatsecplugs Please remove this file, as it is not required for open sourcing or certifying your connector